manifests/mod/security.pp

```diff
@@ -3,6 +3,8 @@
   $activated_rules       = $::apache::params::modsec_default_rules,
   $modsec_dir            = $::apache::params::modsec_dir,
   $modsec_secruleengine  = $::apache::params::modsec_secruleengine,
+  $modsec_secpcrematchlimit = $::apache::params::modsec_secpcrematchlimit,
+  $modsec_secpcrematchlimitrecursion = $::apache::params::modsec_secpcrematchlimitrecursion,
```

Comment:
good question, the parameter-name is long enough without the prefix. I used the prefix because all other  parameter had this prefix too.  i thought it might be some kind of naming convention.